testdata/submoduleAccepted.txt

```diff
@@ -1,3 +1,25 @@
 # Diff files to instead write to submoduleAccepted as "accepted" changes.
 conformance/node10Alternateresult_noTypes.errors.txt.diff
-conformance/node10AlternateResult_noResolution.errors.txt.diff
\ No newline at end of file
+conformance/node10AlternateResult_noResolution.errors.txt.diff
+fourslash/inlayHints/inlayHintsInteractiveParameterNamesWithComments.baseline.diff
```

Comment:
This diff is because I fixed a bug from Strada where we didn't fully detect if a comment had the parameter name.